### PR TITLE
Created a very basic main menu

### PR DESCRIPTION
**Pull request by ALobker**
Removes the blue/red button in the main menu screen and replaces it with 4 buttons

### CODE REVIEW

**Comment by ALobker** [comment]
By the way, use this url to ignore whitespace changes:
[?w=1](https://github.com/iiKoe/bustamove/pull/8/files?w=1)

File: core/src/com/group66/game/screens/MainMenuScreen.java [hunk 1]
```diff
@@ -2,137 +2,240 @@
...
+     * @see com.badlogic.gdx.Screen#render(float)
+     */
+    @Override
+    public void render(float delta) {
+        Gdx.gl.glClearColor(0.2f, 0.2f, 0.3f, 1);
+        Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
+        game.batch.begin();
+
+        /*
+         * if (Gdx.input.getX() < getCenterWidth(PLAY_BUTTON_WIDTH) +
```
Comment (iiKoe) [inline core/src/com/group66/game/screens/MainMenuScreen.java]: If you remove this comment block, i'm more then happy. Rest looks fine

**Comment by iiKoe** [comment]
You also have 3 checkstyle warning, could you fix those too?

Did you check if the reports for findbugs and pmd where OK? (pmd should be clean, findbugs only warnings for now, as those still need fixing)

[ant:checkstyle] /home/travis/build/iiKoe/bustamove/core/src/com/group66/game/screens/GameScreen.java:56:9: warning: Missing a Javadoc comment.
[ant:checkstyle] /home/travis/build/iiKoe/bustamove/core/src/com/group66/game/screens/GameScreen.java:56:42: warning: WhitespaceAround: '{' is not preceded with whitespace.
[ant:checkstyle] /home/travis/build/iiKoe/bustamove/core/src/com/group66/game/screens/MainMenuScreen.java:90: warning: Distance between variable 'exitButton' declaration and its first usage is 4, but allowed 3.  Consider to make that variable as final if you still need to store its value in advance (before method calls that might do side effect on original value).

File: core/src/com/group66/game/screens/MainMenuScreen.java (hunk 1 above)
Comment (jeroenoverman) [inline core/src/com/group66/game/screens/MainMenuScreen.java]: Agree with @iiKoe

**Pull request merged**